mora/integrations/serviceplatformen.py

```diff
@@ -0,0 +1,172 @@
+#
+# Copyright (c) 2017-2018, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import random
+
+from requests import HTTPError
+
+from mora import settings
+
+
+def get_citizen(cpr):
+    if settings.PROD_MODE:
```

Comment:
Shouldn't this appear in `mora-example.json`?

Reply:
It should. I'll fix it.